### DISCUSSION TITLE
Release of Version 3.5.1

### DISCUSSION THREAD

**Discussion by cgiesche** [cgiesche/streamdeck-homeassistant#292] [Announcements]
* Added better support for Encoders
* Added fully customizable, automatically updated rendering configuration

![grafik](https://github.com/user-attachments/assets/501248c4-ccff-48e7-a95d-237783099169)

![grafik](https://github.com/user-attachments/assets/9328c63c-0917-41fa-b633-2ec7dc7f5f9f)

## What's Changed
* #250 Improve Entity selection (list, filter) by @cgiesche in https://github.com/cgiesche/streamdeck-homeassistant/pull/251
* Improved active-states for image buttons. by @cgiesche in https://github.com/cgiesche/streamdeck-homeassistant/pull/249
* Update README.md by @cgiesche in https://github.com/cgiesche/streamdeck-homeassistant/pull/256
* Update PiComponent.vue - minor validation change by @dipsylala in https://github.com/cgiesche/streamdeck-homeassistant/pull/273
* Initial encoder layout support by @cvpcs in https://github.com/cgiesche/streamdeck-homeassistant/pull/278
* Feature/254 improve display image rendering by @cgiesche in https://github.com/cgiesche/streamdeck-homeassistant/pull/291

## New Contributors
* @dipsylala made their first contribution in https://github.com/cgiesche/streamdeck-homeassistant/pull/273
* @cvpcs made their first contribution in https://github.com/cgiesche/streamdeck-homeassistant/pull/278

**Full Changelog**: https://github.com/cgiesche/streamdeck-homeassistant/compare/2.4.5...3.5.1

<hr /><em>This discussion was created from the release <a href='https://github.com/cgiesche/streamdeck-homeassistant/releases/tag/3.5.1'>3.5.1</a>.</em>

**Comment by fkthisusrrequirement** [comment]
Hello,

Is it possible to change the color of each icon? If yes, can you please  provide information as to how?

Thanks
